lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -191,7 +191,11 @@ def database_from_yaml
 
         def database_yaml
           return nil unless File.exist?('config/database.yml')
-          yaml = YAML.load_file('config/database.yml')
+          begin
+            yaml = YAML.load_file('config/database.yml')
+          rescue Psych::SyntaxError
```

Comment:
I'd probably use a method level `rescue` here, so the related code is grouped more closely together.